src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -102,6 +102,12 @@
     <PlatformManifestFileEntry Include="libcoreclr.so" IsNative="true" />
     <PlatformManifestFileEntry Include="libcoreclr.dylib" IsNative="true" />
     <PlatformManifestFileEntry Include="libcoreclr_static.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libcoreclrminipal.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libcoreclrpal.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libeventprovider.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libgc_pal.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libnativeresourcestring.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libminipal.a" IsNative="true" />
```

Comment:
I'd have expected all of these to be contained in `libcoreclr_static.a`. That's what CoreCLR/iOS builds were expecting and it seemed to work.

Reply:
Yup, that's what I was thinking https://github.com/dotnet/runtime/pull/114629#discussion_r2042249448. Keeping the shared fx layout streamlined with other mobile platforms would be good (unless there is a reason to diverge).

Reply:
I think this was because they are static libs, so when we linked them to `coreclr_static`, the objects aren't actually in `coreclr_static` (unlike the various other object libraries we have that do all end up in there) and the final binary linking against `coreclr_static` (`monodroid` in this case) also needed to link against these.

I'd agree that keeping the layout streamlined (that is, only having `coreclr_static`) would be preferable. I'm not sure what the difference is for iOS that it didn't need anything different. For Android, we may want to build object libraries for all these, such that they are directly linked into `coreclr_static`.